api/app/signals/apps/signals/utils/location.py

```diff
@@ -36,7 +36,79 @@ def _get_stadsdeel_code(geometry: PointField, default: Optional[str] = None) ->
     if not settings.FEATURE_FLAGS.get('API_DETERMINE_STADSDEEL_ENABLED', False):
         return default
 
+    from signals.apps.signals.models.location import AREA_STADSDEEL_TRANSLATION
+
     area_type = getattr(settings, 'API_DETERMINE_STADSDEEL_ENABLED_AREA_TYPE', 'sia-stadsdeel')
     area = _get_area(geometry=geometry, area_type=area_type)
     code = AREA_STADSDEEL_TRANSLATION.get(area.code.lower(), None) if area else None
     return code or default
+
+
+class AddressFormatter:
+    re_formatchars = re.compile(r'(?<!\\)([OhltTpPW])')
+    re_escaped = re.compile(r'\\(.)')
```

Comment:
Consider adding a pointer to the Django date formatting functionality that this is based on.

Reply:
Done